README.md

```diff
@@ -30,18 +30,19 @@ var svg = d3.select("#timeline1").append("svg").attr("width", 500)
 
 Works with circles. In case the rectangular edges are too pointy.
 
-![Circular Timeline](https://raw.github.com/jiahuang/d3-timeline/master/examples/timeline2.png
-)
+![Circular Timeline](examples/timeline2.png)
+
+Combine rectangles and circles to your liking.
```

Comment:
Can you add the example dataset in here? Or mention that you can add a "display" key to the data item?